draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1289,27 +1296,30 @@ The following procedure implements hash\_to\_base.
 hash_to_base(msg, ctr)
 
 Parameters:
-- H, a cryptographic hash function producing b bits.
+- DST, a domain separation tag (see discussion above).
+- H, a cryptographic hash function.
 - F, a finite field of characteristic p and order q = p^m.
-- W = ceil((ceil(log2(p)) + k) / b), where k is the security
+- L = ceil((ceil(log2(p)) + k) / 8), where k is the security
   parameter of the cryptosystem (e.g., k = 128).
+- HKDF-Extract and HKDF-Expand are as defined in {{RFC5869}}, 
```

Comment:
Got caught by this last time: hotlinks are broken inside fenced blocks :face_with_head_bandage:

```suggestion
- HKDF-Extract and HKDF-Expand are as defined in RFC5869, 
```